packages/data-explorer/tests/unit/store/getters.spec.ts

```diff
@@ -98,4 +98,66 @@ describe('getters', () => {
       expect(getters.hasEditSettingsRights(state, localGetters)).toEqual(true)
     })
   })
+
+  describe('tableIdAttributeName', () => {
+    let state: any = {}
+
+    it('should return undefined if not set', () => {
+      expect(getters.tableIdAttributeName(state)).toEqual(undefined)
+    })
+
+    it('should return name of the id field if set', () => {
+      state = {
+        tableMeta: {
+          idAttribute: {
+            name: 'some-name'
+          }
+        }
+      }
+      expect(getters.tableIdAttributeName(state)).toEqual('some-name')
+    })
+  })
+
+  describe('tableLabelAttributeName', () => {
+    let state: any = {}
+
+    it('should return undefined if not set', () => {
+      expect(getters.tableLabelAttributeName(state)).toEqual(undefined)
+    })
+
+    it('should return name of the label field if set', () => {
+      state = {
+        tableMeta: {
+          labelAttribute: {
+            name: 'some-label'
```

Comment:
Just a strange linting note:
Why is 'labelAttribute' on line 132 stretched over 3 lines, and line 150 and 159 with two properties on one line?